.markdownlint.yaml

```diff
@@ -0,0 +1,9 @@
+MD001: false
+
+MD010: false
+
+MD013:
+  code_blocks: false
+  tables: false
+
+MD033: false
```

Comment:
These disabled rules might sound opinionated, mainly because they come without a comment about what they are about and why they are deactivated.

I know markdown lint, and I'm OK with the rules you chose, but I think comments would help

Reply:
Good point, we definitely should add a summary about what they are disabling for each one at the very least. 

Feel free to do it if you like, if not I’ll slide into my day Monday

Cheers

Reply:
I could, but I'm not in your head and the reason why you removed might differ from what I think you removed them 😅

Reply:
I opened #143

Reply:
Awesome thanks, I’ll add you to review Monday.